tracker/templates/base.html

```diff
@@ -47,5 +47,29 @@
 			{%- endwith -%}
 			{%- block content -%}{%- endblock %}
 		</div>
+		<div id="page_footer">
+        	<p>
+				Copyright © 2002-2021 
```

Comment:
This line and the line below contains an extraneous space at the end of the line